### PR TITLE
Bugfix/sfw 204 jazzy docs

### PR DESCRIPTION
**Pull request by ronanociosoig**
Docs are now working and they have 71% coverage. 

Updated all usage of "enable" to "enabled".

### CODE REVIEW

File: StanwoodAnalytics/Classes/Core/StanwoodAnalytics.swift [hunk 1]
```diff
@@ -412,24 +343,34 @@ open class StanwoodAnalytics {
...
+        */
         public func build() -> StanwoodAnalytics {
             return StanwoodAnalytics(builder: self)
         }
-        
-        public func setExceptionTracking(enable: Bool) {
-            
-        }
+ 
+        // public func setExceptionTracking(enable: Bool) {
```
Comment (talezion) [inline StanwoodAnalytics/Classes/Core/StanwoodAnalytics.swift:372]: @ronanociosoig do we need this?

File: StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift [hunk 2]
```diff
@@ -87,7 +98,11 @@ open class BugfenderTracker: Tracker {
                       message: trackingParameters.description ?? "")
     }
     
-    override open func setTracking(enable: Bool) {
+    
+    /// Set Tracking. Not implemented.
+    ///
+    /// - Parameter enabled: Enabled
```
Comment (talezion) [inline StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift:104]: @ronanociosoig `Enabled = Bool`?

File: StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift [hunk 3]
```diff
@@ -119,39 +133,48 @@ open class BugfenderTracker: Tracker {
...
+//    }
+//
+//    static func log(lineNumber: Int?, method: String?, file: String?, level: BugfenderType, tag: String?, message: String) {
+//        print("Line: \(String(describing:lineNumber))")
+//        print("Method: \(String(describing:method))")
+//        print("File: \(String(describing:file))")
+//        print("Tag: \(String(describing:tag))")
+//        print("Message: \(String(describing:message))")
+//    }
+//}
```
Comment (talezion) [inline StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift:179]: @ronanociosoig Is this still required?

**Review by talezion — APPROVED** [review]
Added a few comments

File: StanwoodAnalytics/Classes/Core/StanwoodAnalytics.swift (hunk 1 above)
Comment (ronanociosoig) [reply StanwoodAnalytics/Classes/Core/StanwoodAnalytics.swift:372]: Removed.

File: StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift (hunk 3 above)
Comment (ronanociosoig) [reply StanwoodAnalytics/Classes/SubSpecs/BugFender/BugfenderTracker.swift:179]: removed.

**Pull request merged**